src/gui/widgets/chatbox.cpp

```diff
@@ -89,14 +89,23 @@ void chatbox::finalize_setup()
 
 void chatbox::load_log(std::map<std::string, chatroom_log>& log, bool show_lobby)
 {
-	for(const auto& l : log) {
+	const std::string new_tip = formatter()
+		<< "\n"
+		<< markup::span_color("#FF0000", "============" + _("NEW") + "============");
```

Comment:
In a new line immediately preceding this one, add `# po: <your translation hint>` here.

Reply:
@Wedge009 that works for WML files but not C++ ones.
For c++ its `// TRANSLATORS <your po hint>`

Reply:
Ah, you're right, thanks. Been working with Python more than C++ lately so I forgot the syntax was different.

Reply:
`// TRANSLATORS: This is the new chat text indicator` is sufficient.

Reply:
```suggestion
		markup::span_color("#FF0000", "============", _("NEW"), "============");
```

Reply:
Didn't know you could do that. Agreed this way is better.